ssl/statem/extensions.c

```diff
@@ -1833,6 +2136,9 @@ static EXT_RETURN tls_construct_compress_certificate(SSL_CONNECTION *sc, WPACKET
 
     if (sc->cert_comp_prefs[0] == TLSEXT_comp_cert_none)
         return EXT_RETURN_NOT_SENT;
+# ifndef OPENSSL_NO_ECH
```

Comment:
Instead of # if guards around each call to `ECH_SAME_EXT(sc, pkt)`, I will define a empty macro instead o current definition , when OPENSSL_NO_ECH is defined.
Ate the end, less lines changed.

Reply:
Hmm - is there any precedent for that kind of thing? If not, I think I prefer the way it is now as (I think:-) it's clearer to the reader of the code. But you're right that it could be done that way, so I'd not object to changing if others also considered that preferable.

Reply:
OK to keep as is but also to change @sftcd it is up to you.